data/json/mapgen_palettes/house_general_palette.json

```diff
@@ -160,7 +188,6 @@
       ":": "t_wall_glass",
       "-": "t_sidewalk",
       "o": [
-        [ "t_window_frame", 4 ],
```

Comment:
Recommend keeping this but toning down to  [ "t_window_frame", 2 ],

Reply:
Personally I don't understand why you want there to be consistent damage across all buildings as opposed to variation, like I said if you want to make a PR arguing for this go ahead but I don't agree. Making pristine buildings rarer would be fine but like I said I want to wait for player feedback. Equally this is representing the starting day situation, I'd be down to reduce pristine variants over time once I get round to using and expanding on #66890